src/actions/party.ts

```diff
@@ -2,115 +2,79 @@ import { ActionTypes } from './types';
 import Party from '../services/party';
 import { NavigationActions } from 'react-navigation';
 
-export class PartyActions {
```

Comment:
Restoring this code as it was before because it's a little bit harder to make the `bindActionCreators` with a Object containing other `ActionCreators`